lib/views/mnemonic_onboarding.dart

```diff
@@ -44,7 +44,7 @@ class _MnemonicOnboardingState extends State<MnemonicOnboarding> {
                       ),
                       SizedBox(height: 16),
                       Text(
-                        'Please write this down somewhere safe, if you lose this, you will not get your account back. Not even Demeris could help you with that.',
+                        'Follow best practices for security. Be sure to write your mnemonic pass phrase in a safe place. This phrase is the only way to recover your account if you forget your password. Without your password or recovery passphrase, account recovery is not possible. Even Emeris cannot help you.',
```

Comment:
@wal33d006 let's be clear about what we are asking the user to write down 
ambiguous pronoun references happen all the time, especially in code comments and technical docs 

see https://developers.google.com/style/pronouns 
Key Point: Ensure that a pronoun clearly refers to its antecedent (the noun that it's replacing).

https://developers.google.com/style/word-list#letter-t
the "this" in "please write this down" was unclear to me (my editor calls this an unclear reference, (our docs have a lot of those "opportunities" for clarity)

this, that
Where possible, put a noun after this or that for clarity. If doing so results in clunky prose, then don't do it; but even then, try thinking about what the noun would be. If you aren't sure what noun this or that refers to, then consider rephrasing.

Reply:
Awesome! That's amazing observation! Thanks for the update!

Reply:
@wal33d006 so my guess was correct? we're asking the user to write down their mnemonic pass phrase?

Reply:
@barriebyron 
Correct. We are asking the user to write this mnemonic passphrase in a secure place